tests/v2/validation/rbac/psa/psa.go

```diff
@@ -137,19 +75,6 @@ func createDeploymentAndWait(steveclient *v1.Client, client *rancher.Client, clu
 	return deploymentResp, err
 }
 
-func getAndConvertNamespace(namespace *v1.SteveAPIObject, steveAdminClient *v1.Client) (*coreV1.Namespace, error) {
-	getNSSteveObject, err := steveAdminClient.SteveType(namespaces.NamespaceSteveType).ByID(namespace.ID)
-	if err != nil {
-		return nil, err
-	}
-	namespaceObj := &coreV1.Namespace{}
-	err = v1.ConvertToK8sType(getNSSteveObject.JSONResp, namespaceObj)
-	if err != nil {
-		return nil, err
-	}
-	return namespaceObj, nil
-}
-
 func deleteLabels(labels map[string]string) {
```

Comment:
Can this be a extension since this seems generic and not specific to PSA?

Reply:
Delete labels in this file is specific to PSA. This is what the body contains:

`		if strings.Contains(label, psaWarn) || strings.Contains(label, psaAudit) || strings.Contains(label, psaEnforce) {
			delete(labels, label)`

It checks for psa labels and deletes them

Reply:
Sorry, missed that! Can we rename this function to deletePsaLabels?